frontend/src/components/dashboard/DataDashboard.vue

```diff
@@ -1,5 +1,6 @@
 <script setup lang="ts">
 import Demo from './data/Demo.vue'
+import IPEDS_318_45 from './data/IPEDS_318_45.vue'
```

Comment:
Just to confirm: this file doesn't exist yet, right? I know this is just a draft PR.

Reply:
fixed